src/components/ens-registration/RegistrationAvatar/RegistrationAvatar.tsx

```diff
@@ -69,13 +77,8 @@ const RegistrationAvatar = ({
 
   const accentColor = useForegroundColor('accent');
 
-  const { ContextMenu } = useSelectImageMenu({
-    imagePickerOptions: {
-      cropperCircleOverlay: true,
-      cropping: true,
-    },
-    menuItems: ['library', 'nft'],
-    onChangeImage: ({
+  const onChangeImage = useCallback(
```

Comment:
how come this is memoized now?

Reply:
i needed to use it here https://github.com/rainbow-me/rainbow/pull/3320/files#diff-979c9a5a3bd5e6c060d240887a5c189b4b70b4198d86222ce45e89c2e4a44478R143 to be able to press the avatar and go directly to the NFT selection sheet to skip context menus because of bitrise failing in that step